guacamole-docker/README.md

```diff
@@ -129,17 +126,17 @@ Deploying Aoache Guacamole with LDAP authentication
         -e LDAP_CONFIG_BASE_DN=ou=connections,dc=example,dc=com \
         -d -p 8080:8080 glyptodon/guacamole
 
-Linking Guacamole to your LDAP directory will require additional configuration parameters
+Using Guacamole with your LDAP directory will require additional configuration parameters
 specified via environment variables. These variables collectively describe how Guacamole
 will connect to LDAP:
 
-1. `LDAP_HOSTNAME` - The base of the DN for all Guacamole users.
+1. `LDAP_HOSTNAME` - The hostname or IP address of your LDAP server.
 2. `LDAP_USER_BASE_DN` - The base of the DN for all Guacamole users.
 3. `LDAP_PORT` - The port your LDAP server listens on.  (Optional)
```

Comment:
Hm... What do you think about listing the optional variables in their own list? I think this is how things are organized in the manual.